staging/src/k8s.io/apiserver/pkg/storage/etcd3/logger.go

```diff
@@ -32,15 +32,15 @@ type klogWrapper struct{}
 const klogWrapperDepth = 4
 
 func (klogWrapper) Info(args ...interface{}) {
-	klog.InfoDepth(klogWrapperDepth, args...)
+	//Discard all Info logging. Do nothing.
 }
 
 func (klogWrapper) Infoln(args ...interface{}) {
-	klog.InfoDepth(klogWrapperDepth, fmt.Sprintln(args...))
+	//Discard all Info logging. Do nothing.
 }
 
 func (klogWrapper) Infof(format string, args ...interface{}) {
-	klog.InfoDepth(klogWrapperDepth, fmt.Sprintf(format, args...))
+	//Discard all Info logging. Do nothing.
```

Comment:
Depending on verbosity level we still want these. Maybe at 9 ? Don't know how we map that elsewhere.

Reply:
Or maybe 4 or 6.

Reply:
Agree, seems needless to totally throw this away... can we not bump it to a more verbose log level?